openvdb/openvdb/tools/Count.h

```diff
@@ -62,6 +63,10 @@ Index64 countActiveTiles(const TreeT& tree, bool threaded = true);
 template <typename TreeT>
 Index64 memUsage(const TreeT& tree, bool threaded = true);
 
+/// @brief Return the minimum and maximum active values in this tree.
+template <typename TreeT>
+math::MinMax<typename TreeT::ValueType> minMax(const TreeT& tree, bool threaded = true);
```

Comment:
minor: a comment that this returns zeroVal<T> (i think?) for empty trees would be great